PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py

```diff
@@ -44,14 +44,19 @@
     embedTrack                  = cms.bool(True),
     )
 
+# Rerun IDProducer on pat::Electrons
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronID_cfi import *
+patLowPtGsfElectronID = lowPtGsfElectronID.clone(electrons='patLowPtElectrons',
+                                                 rho='fixedGridRhoFastjetAll')
+
 makePatLowPtElectronsTask = cms.Task(
     lowPtElectronMatch,
-    patLowPtElectrons
+    patLowPtElectrons,
+    patLowPtGsfElectronID
     )
 
```

Comment:
```suggestion
    from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
    run2_miniAOD_UL.toReplaceWith(makePatLowPtElectronsTask, cms.Task(makePatLowPtElectronsTask.copy()+lowPtGsfElectronID))
```
this would enable re-running of lowPtGsfElectronID only for specific inputs

Reply:
I'm not sure what you mean by this comment.

I want to rerun both the ID and the rekey modules for run2_miniAOD_UL.
Is [this](https://github.com/cms-sw/cmssw/blob/a86c00fae073865dc12bb6c4a75f9d48f9f86671/PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py#L90-L96) not correct?

Reply:
the new version seems OK, the comment was made before rekeying was in the config